playwright/tests/vamcSystem.spec.js

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '../utils/next-test'
+
+// This test should be modified to test page output and function.
+// test.describe('Vamc_system', () => {
+test.skip('Vamc_system', () => {
```

Comment:
Should I bother with the playwright tests yet?

Reply:
🤷 Probably would be good to eventually, but I haven't done this yet either. You definitely don't need to worry about it for now.